tensorrt_llm/_torch/models/checkpoints/mx/checkpoint_loader.py

```
@@ -0,0 +1,378 @@
+        prior = {key: os.environ.get(key) for key in env_overrides}
+        for key, value in env_overrides.items():
+            os.environ[key] = value
+
+        try:
+            publish_model_params(model)
+            logger.info(
+                "Published weights to MX server at %s as model=%r",
+                self._mx_server_url,
+                resolved_name,
+            )
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)